### PR TITLE
fetch full names of users

### PR DESCRIPTION
**Pull request by krisbitney**
Adds support for fetching users' full names from GoodDollar's mongodb database

### CODE REVIEW

**Comment by krisbitney** [comment]
@sirpy @L03TJ3 I don't have a way to test whether the fetch from mongodb is working. Could you please assign a full name to a supporter or steward's address in your database and tell me the address?

You can use my address if you'd like: `0xC3C54Ee8B495A5004602AFf40c5880fb8E375aaD`.

**Comment by krisbitney** [comment]
I just remembered that I forgot to add caching. I will add that tomorrow.

**Review by sirpy — CHANGES_REQUESTED** [review]
I've created here a full example of how to use mongodb with graphql.
it also includes caching policies package.
and a modified fetch query to fetch all profiles at once instead of doing multiple queries.
https://github.com/sirpy/realm-graphql-apollo-react
index.js - shows how client and cache are initialized
graphql-operations.js - includes the graphql query
app.js - the fetch query itself

create a separate graphql client for profile queries

File: packages/app/src/hooks/apollo/useCreateMongoDbApolloClient.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            (options.headers as Record<string, any>).Authorization = `Bearer ${accessToken}`;
+            return fetch(uri, options);
+          },
+        }),
+        defaultOptions: {
+          watchQuery: {
+            fetchPolicy: 'no-cache',
+          },
+          query: {
+            fetchPolicy: 'no-cache',
```
Comment (sirpy) [inline packages/app/src/hooks/apollo/useCreateMongoDbApolloClient.ts:65]: cache-and-network

**Review by sirpy — APPROVED** [review]

File: packages/app/src/hooks/apollo/useCreateMongoDbApolloClient.ts (hunk 1 above)
Comment (krisbitney) [reply packages/app/src/hooks/apollo/useCreateMongoDbApolloClient.ts:65]: Sure. I literally copy-pasted the 'no-cache' from the example you requested I implement.

Comment (krisbitney) [reply packages/app/src/hooks/apollo/useCreateMongoDbApolloClient.ts]: this is changed in a commit

**Comment by krisbitney** [comment]
@sirpy I am getting the following error. Is this the issue of needing to whitelist a URL? The same as before?
Screenshot 2024-01-18 at 1 57 12 PM

**Pull request merged**